multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/util/DeprecatedBuildpackChecker.java

```diff
@@ -0,0 +1,61 @@
+package org.cloudfoundry.multiapps.controller.process.util;
+
+import java.util.List;
+import java.util.Map;
+
+import jakarta.inject.Named;
+import org.cloudfoundry.multiapps.controller.core.helpers.ModuleToDeployHelper;
+import org.cloudfoundry.multiapps.controller.core.model.SupportedParameters;
+import org.cloudfoundry.multiapps.mta.builders.v2.ParametersChainBuilder;
+import org.cloudfoundry.multiapps.mta.model.DeploymentDescriptor;
+import org.cloudfoundry.multiapps.mta.model.Module;
+import org.cloudfoundry.multiapps.mta.util.PropertiesUtil;
+
+@Named
+public class DeprecatedBuildpackChecker {
+
+    private static final String DEPRECATION_MESSAGE = "SAP Java Buildpack 1 has been deprecated and is going to be removed from SAP BTP, Cloud Foundry environment on June 30, 2025!";
```

Comment:
maybe add an additional sentence about how this is currently the default so the user isn't confused if the module is simply defined as 'java.tomee' or 'java'. And modify the message a bit if the current date is past June 30?